src/mlpack/tests/ann_layer_test.cpp

```diff
@@ -1004,5 +1004,26 @@ BOOST_AUTO_TEST_CASE(SimpleMeanSquaredErrorLayerTest)
   BOOST_REQUIRE_EQUAL(output.n_elem, 1);
 }
 
+/*
+ * Simple test for the Resize layer
+ */
+
+BOOST_AUTO_TEST_CASE(SimpleResizeLayerTest)
```

Comment:
I suggest a more meaningful test:
1. Declare a n*m matrix.
2. Fill the matrix using a linear function (e.g. `x+y` or `ax+by`)
3. Scale the matrix.
4. Check if the scaled matrix represents the scaled linear function.
5. Transform the scaled matrix backward.
6. Check if this matrix is close to the original one.